src/utils/automate_steps.py

```diff
@@ -36,7 +36,7 @@ def resume(self):
 
     def pause(self):
         if self.thread:
-            self.thread.is_run = False
+            self.thread.is_running = False
```

Comment:
`Runner` still has `is_run` field, not `is_running, so the pausing doesn't work
```suggestion
            self.thread.is_run = False
```